release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -2110,7 +2110,24 @@ issue by deleting the Elasticsearch deployments in sequence.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1824006[*BZ#1824006*])
 
 * {product-title} 4.4 is not shipping with a v.4.4 Metering Operator. Customers can install or continue to run the v.4.3 Metering Operator on {product-title} 4.4 clusters.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1829035[*BZ1829035*])
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1829035[*BZ#1829035*])
+
+* When updating an {product-title} cluster from version 4.3 to 4.4, the etcd
+Operator fails to upgrade. As a workaround, complete the following steps:
++
+. Force a new revision on etcd:
++
+----
+$ oc patch etcd cluster -p='{"spec": {"forceRedeploymentReason": "recovery-'"$( date --rfc-3339=ns )"'"}}' --type=merge
+----
++
+. Verify the nodes are at the latest revision:
++
+----
+$ oc get etcd '-o=jsonpath={range .items[0].status.conditions[?(@.type=="NodeInstallerProgressing")]}{.reason}{"\n"}{.message}{"\n"}'
+----
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1830409[*BZ#1830409*])
```

Comment:
We now have a 4.4.z clone in https://bugzilla.redhat.com/show_bug.cgi?id=1830789 that these 4.4 notes should probably link instead.